src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -48,11 +49,12 @@
 import org.elasticsearch.client.Client;
 import org.elasticsearch.common.logging.ESLogger;
 import org.elasticsearch.common.logging.Loggers;
+import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.index.IndexNotFoundException;
 import org.elasticsearch.search.SearchHit;
 import org.elasticsearch.transport.RemoteTransportException;
 
-public class KibanaSeed {
+public class KibanaSeed implements ConfigurationSettings {
```

Comment:
Why do we need to impl ConfigurationSettings?  I didnt see any obvious uses of it.